src/main/java/be/seeseemelk/mockbukkit/WorldMock.java

```diff
@@ -68,6 +68,7 @@
 import be.seeseemelk.mockbukkit.entity.ItemEntityMock;
 import be.seeseemelk.mockbukkit.entity.ZombieMock;
 import be.seeseemelk.mockbukkit.metadata.MetadataTable;
+import org.jetbrains.annotations.Nullable;
```

Comment:
This import was already imported. No need to import it twice
```suggestion
```

Reply:
Should be resolved by 69b612b7f159be6b166906cef5479244eff729b4